### ISSUE TITLE
Latest release 1.1.0 not working with Netbox 2.10

### ISSUE THREAD

**Issue by kbcz1989** [marcus-crane/netbox-plugin-azuread#9]
Hello @marcus-crane.

Thank you for this great plugin. I was just testing today when I realized that the latest update is not working on Netbox 2.10.

```django
Server Error

There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'django.template.exceptions.TemplateDoesNotExist'>

base/base.html

Python version: 3.8.7
NetBox version: 2.10.4

If further assistance is required, please post to the NetBox mailing list. 
```

**Comment by marcus-crane** [comment]
Hey @kbcz1989,

Apologies for the delay with this issue, although nothing has substantially changed with the plugin so it's not like you were missing out on anything!

Anyway, I've just released [v1.1.2](https://github.com/marcus-crane/netbox-plugin-azuread/releases/tag/1.1.2) which dynamically renders either a 2.x compatible template (just the original template) or a 3.x compatible template (the new fancy one) depending on your Netbox version.

I wanted to get this one out of the way since I just added some verbose debug logging (v1.1.1) and didn't want 2.x users to miss out on new functionality.
